types.d.ts

```diff
@@ -0,0 +1,29 @@
+// https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68600
+declare module 'react/jsx-runtime' {
+  import {ElementType, Fragment, Key, ReactElement} from 'react'
+
+  function jsx(type: ElementType, props: unknown, key?: Key): ReactElement
+
+  export {Fragment, jsx, jsx as jsxs}
+}
+
+// https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68600
+declare module 'react/jsx-dev-runtime' {
+  import {ElementType, Fragment, Key, ReactElement} from 'react'
+  import {Source} from 'hast-util-to-jsx-runtime'
+
+  function jsxDEV(
+    type: ElementType,
+    props: unknown,
+    key: Key | undefined,
+    isStatic: boolean,
+    source?: Source,
+    self?: unknown
+  ): ReactElement
+
+  export {Fragment, jsxDEV}
+}
+
+declare module 'https://esm.sh/hastscript@8?dev' {
+  export * from 'hastscript'
+}
```

Comment:
See https://github.com/syntax-tree/hast-util-to-jsx-runtime/pull/5

I changed the comment for https://esm.sh, because there is no indication TypeScript will add support for this.